gcalcli/gcal.py

```diff
@@ -1424,25 +1424,17 @@ def CreateEventFromVOBJ(ve):
                     print('Location.....%s' % ve.location.value)
                 event['location'] = ve.location.value
 
-            if not hasattr(ve, 'dtstart') or not hasattr(ve, 'dtend'):
+            if not hasattr(ve, 'dtstart'):
                 self.printer.err_msg(
-                        'Error: event does not have a dtstart and dtend!\n'
+                    'Error: event does not have a dtstart!\n'
                 )
                 return None
 
-            if verbose:
-                if ve.dtstart.value:
-                    print('Start........%s' % ve.dtstart.value.isoformat())
-                if ve.dtend.value:
-                    print('End..........%s' % ve.dtend.value.isoformat())
-                if ve.dtstart.value:
-                    print('Local Start..%s' %
-                          self._localize_datetime(ve.dtstart.value)
-                          )
-                if ve.dtend.value:
-                    print('Local End....%s' %
-                          self._localize_datetime(ve.dtend.value)
-                          )
+            #if not hasattr(ve, 'dtend') and not hasattr(ve, 'duration'):
```

Comment:
I don't think we want to merge changes that comment out blocks.  We should just delete unused code.